src/main/resources/db/migration/mysql/V1.1.5__add-event-learner-booking-tables.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE `event` (
+  `id` bigint NOT NULL AUTO_INCREMENT,
+  `path` varchar(255) NOT NULL,
+  PRIMARY KEY(`id`)
+);
+
+CREATE TABLE `learner` (
+  `id` bigint NOT NULL AUTO_INCREMENT,
+  `uuid` varchar(60) NOT NULL,
+  PRIMARY KEY(`id`)
+);
+
+CREATE TABLE `booking` (
+  `id` bigint NOT NULL AUTO_INCREMENT,
+  `learner_id` bigint NOT NULL,
+  `event_id` bigint NOT NULL,
+  `payment_details` varchar(255),
+  `status` varchar(9) NOT NULL,
+  `booking_time` datetime NOT NULL,
```

Comment:
can we order them in this state please?
```
booking_time,
status,
payment_details
```

Status_type check shouldn't be ENUM and all UPPERCASE?